src/ocgis/conv/base.py

```diff
@@ -148,15 +148,16 @@ def _get_return_(self):
     def _write_coll_(self, f, coll):
         raise NotImplementedError
 
-    def _finalize_(self, *args, **kwargs):
-        raise NotImplementedError
-
     def _get_or_create_shp_folder_(self):
         path = os.path.join(self.outdir, 'shp')
         if not os.path.exists(path):
             os.mkdir(path)
         return path
 
+    def _preformatting(self, i, coll):
```

Comment:
Should name this ``_preformatting_`` since it is a private method.

Reply:
Fixed.